tools/test/travis-ci/scancode_test/scancode_test_1.json

```
@@ -0,0 +1,7 @@
+{
+    "headers": [
+      {
+        "tool_name": "scancode test fail"
+      }
+    ]
+  }
```

Create this file on the fly.
Also the indentation is off.
```suggestion
{
  "headers": [
    {
      "tool_name": "scancode test fail"
    }
  ]
}
```